src/descriptor_runner/descriptor_runner/descriptor_runner_fallback.ts

```diff
@@ -111,10 +113,35 @@ export default class DescriptorRunnerFallback extends DescriptorRunner<GraphDesc
     private async compile(): Promise<void> {
         if (!this.descriptor) throw new Error('Descriptor is not loaded');
 
-        let dnn_fallback_kernel: any = null;
-        eval(this.descriptor.kernel_source);
+        let local_dnn_fallback_kernel = null;
+
+        function loadScript(url){
+            var script = <any>document.createElement("script")
+            script.type = "text/javascript";
+            let promise = new Promise((resolve, reject)=>{
+               if (script.readyState){  //IE
+                    script.onreadystatechange = function(){
+                        if (script.readyState == "loaded" ||
+                                script.readyState == "complete"){
+                            script.onreadystatechange = null;
+                            resolve();
+                        }
+                    };
+                } else {  //Others
+                    script.onload = function(){
+                        resolve();
+                    };
+                }
+            })
+
+            script.src = url;
```

Comment:
use `trasnformUrl(url)` function in `fetch.ts` for transforming url into custom path.

Reply:
What's trasnformUrl means? I do not understand.

Reply:
[`transformUrl(url)` function defined in `fetch.ts`](https://github.com/mil-tokyo/webdnn/blob/master/src/descriptor_runner/fetch.ts#L26). As default, WebDNN loads files of pre-defined urls such as `"${directory}/kernels_fallback.js"`. `transformUrl(url)` supports to transform this url into user-defined custom url.